src/app/components/contribution/contribution.component.html

```diff
@@ -19,13 +19,129 @@
       <p class="header">Interested in <span>Contributing</span>?</p>
     </div>
   </div>
-  <div class="row justify-content-md-center mt-4 text-center">
+  <div class="row justify-content-md-center mt-4 mb-5 text-center">
     <div class="row m-0">
       <div class="cb-github">
         <img src="../../../assets/home/github.gif" alt="github">
       </div>
     </div>
   </div>
+
```

Comment:
Hey @urvashi-code1255 instead of manually making all the contributors list, just use  the snippet I gave below for getting the contributor list and remove all the unnecessary codes 
```
 <a href="https://github.com/Canvasbird/canvasboard/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=Canvasbird/canvasboard" />
</a>
```

Reply:
@Kajol-Kumari Can you please tell me how you got this code template?

Reply:
@urvashi-code1255 you can get this url from [this](https://contrib.rocks/preview) website

Reply:
@Kajol-Kumari I have seen using this website to add contributors image in the readme.md but if we will add it to the website then because it is an image it will ruin the responsiveness of the website. Isn't it?

Reply:
No, I handled it ☺